docs-asciidoc/shared-content/modules/ROOT/pages/working.adoc

```diff
@@ -1,4 +1,4 @@
-Cloudstate provides a good developer experience by implementing the low level details that allow you to focus on your business logic. When using Cloudstate open source, you need to provide your own Kubernetes environment and provision Cloudstate to run in it. Lightbend also provides a https://lbcs.io[paid Cloudstate service] that manages the Kubernetes environment for you. 
+Cloudstate provides a good developer experience by implementing the low level details that allow you to focus on your business logic. When using Cloudstate open source, you need to provide your own Kubernetes environment and provision Cloudstate to run in it. Lightbend also provides a https://lbcs.io[paid Cloudstate service] that manages the Kubernetes environment for you.
```

Comment:
how differs lbcs.io from console.cloudstate.com? Are those instances the same? I meant the latter is the "production" one but I'm not sure.

Reply:
> @marcellanz, good catch! These links should be to console.cloudstate.com.
This was written before that site was live. Also, maybe maybe we should
change "paid Cloudstate service" to just "Cloudstate service", the name
will be changing in the future as well.

Rather than console.cloudstate.com (which will just be the login page), how about we use https://www.lightbend.com/cloudstate-by-lightbend? (Actually, that already redirects to https://www.lightbend.com/akka-serverless)

Reply:
I've updated to the informational link on the Lightbend site, which is now the Akka Serverless page. I've left the link name as "Cloudstate service".